go.mod

```diff
@@ -4,23 +4,26 @@ go 1.12
 
 require (
 	github.com/ONSdigital/dp-api-clients-go v0.0.0-20190920133223-0b75bbb235dd
+	github.com/ONSdigital/dp-authorisation v0.0.0-20191018110224-d4543527d6cb // indirect
```

Comment:
Once you have made all other code changes, if you run `go mod tidy` that way go.mod go.sum will be cleaned up for you

Reply:
Done in commit ""Passing context values into methods rather than instantiating new values each time"